challenge-manager/challenge-tree/paths.go

```diff
@@ -80,27 +80,26 @@ type isConfirmableArgs struct {
 // or non-essential.
 //
 // After the paths are computed, we then compute the path weight of each
-// and insert each weight into a min-heap. If the min element of this heap
-// has a weight >= the confirmation threshold, the
-// essential node is then confirmable.
+// and if the min element of this list has a weight >= the confirmation threshold,
+// the essential node is then confirmable.
```

Comment:
If we do call it `EssentialEdgeId`, let's also update this doc comment to say, `essential edge`